test/functional/outputs/forward_to_kafka_test.log

```diff
@@ -0,0 +1,175 @@
+package outputs
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+        "github.com/openshift/cluster-logging-operator/test/functional"
+	"github.com/openshift/cluster-logging-operator/test"
+	//. "github.com/openshift/cluster-logging-operator/test/matchers"
+)
+
+var _ = Describe("[LogForwarding][Kafka] Functional tests", func() {
+
+	var (
+		framework *functional.FluentdFunctionalFramework
+	)
+
+	BeforeEach(func() {
+		framework = functional.NewFluentdFunctionalFramework()
+	})
+	AfterEach(func() {
+		framework.Cleanup()
+	})
+
+//What would be other Kafka specific values ?
+	setKafkaSpecValues := func(outspec *logging.OutputSpec) {
+		outspec.Kafka = &logging.Kafka{
+			Topic:  kafka.AppLogsTopic,
+		}
+	}
+
+
+	timestamp := "2013-03-28T14:36:03.243000+00:00"
+
+	Context("Application Logs", func() {
```

Comment:
I do not believe we need to ship more then one logtype unless you find logic in the kafka config that is specific to a given log type.  Maybe there is a need for multiple types to ensure we configure multiple topics  correctly?